groups/validators.py

```diff
@@ -0,0 +1,13 @@
+from django.utils import timezone
+
+
+def dates_validator(self, start, end, not_in_past, name, field, items=1):
```

Comment:
1. There is too many local variables, can you remove some variables?
2. What does self variable mean in your method?
3. Why your items variable by default equal 1?
4. What start, end, name variables mean?
5. Why your method returns nothing, maybe you can return cleaned_data here?